Tests/BSGFilepathTests.m

```diff
@@ -0,0 +1,50 @@
+//
+//  BSGFilepathTests.m
+//  Tests
+//
+//  Created by Delisa on 2/19/19.
+//  Copyright © 2019 Bugsnag. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+#include <string.h>
+
+
+int bsg_create_filepath(char *base, char filepath[512], char severity, char error_class[21]);
+
+@interface BSGFilepathTests : XCTestCase
+
+@end
+
+@implementation BSGFilepathTests
+
```

Comment:
It'd be good to verify this method gracefully handles a empty/null errorClass (and possibly other params)